.github/workflows/push.yml

```diff
@@ -42,7 +43,7 @@ jobs:
         cp -f .env.example .env
 
     - name: Checks for new endpoints against AWS WAF rules
-      uses: cds-snc/notification-utils/.github/actions/waffles@master
+      uses: cds-snc/notification-utils/.github/actions/waffles@main
```

Comment:
This probably needs to stay at master if we're referencing `notification-utils`? Thoughts @jimleroyer ?

Reply:
I have created `main` https://github.com/cds-snc/notification-utils/tree/main

Reply:
I guess it will be good then. 👍